pkg/userop/blockchain_backend.go

```diff
@@ -41,6 +41,7 @@ type EthBackend interface {
 	BlockNumber(ctx context.Context) (uint64, error)
 	WaitMinedPeriod() time.Duration
 	RPC() *rpc.Client
+	EthClient() *ethclient.Client
```

Comment:
Why?

Reply:
So that we can get the underlying `ethclient.Client` by `node.EthClient()`.

Reply:
13f58fcb82df6c115de03edabddfd0ed4f298d52